### PR TITLE
Update events.json

### PR DESCRIPTION
**Pull request by LadyAbigor**

### CODE REVIEW

File: app/config/factionhall/events.json [hunk 1]
```diff
@@ -1,60 +1,52 @@
...
-            "description": "Join Lucien Greaves to discuss The Politics of Moral Panic: How Polarization Undermines Collective Reason.",
+            "name": "Sober Faction Memorial Ritual",
+            "description": "Co-Director Priest Joe Dee & Reverend Jon Eldritch with Sober Faction's Council will kick off the event with a memorial ritual for all those lost to overdose.",
             "room": "auditorium",
-            "start": "2022-08-20T20:00:00-04:00",
-            "end": "2022-08-20T21:00:00-04:00",
-            "image": "https://tstvhq.s3.amazonaws.com/backgrounds/factionhall/greyfaction/calendarhall.png"
+            "start": "2022-08-31T20:00:00-04:00",
+            "end": "2022-08-31T21:00:00-04:00",
+            "image": "https://tstvhq.s3.amazonaws.com/backgrounds/factionhall/Sober+Faction+2022/SF+CALENDAR.jpeg"
```
Comment (kacycrider) [inline app/config/factionhall/events.json]: This isn't going to be legible

File: app/config/factionhall/events.json [hunk 2]
```diff
@@ -1,60 +1,52 @@
...
-            "room": "hall",
-            "start": "2022-08-20T22:00:00-04:00",
-            "end": "2022-08-20T23:00:00-04:00",
-            "image": "https://tstvhq.s3.amazonaws.com/backgrounds/factionhall/greyfaction/calendarhall.png"
+            "name": "Open Share Sober Faction Meeting",
+            "description": "An open share Sober Faction meeting will take place for all to remember loved ones.",
+            "room": "large-lounge",
+            "start": "2022-08-31T22:00:00-04:00",
+            "end": "2022-08-31T23:00:00-04:00",
+            "image": "https://tstvhq.s3.amazonaws.com/backgrounds/factionhall/Sober+Faction+2022/SF+CALENDAR.jpeg"
```
Comment (kacycrider) [inline app/config/factionhall/events.json]: If there are issues with this, I'd blame the spaces left in the filenames that are replaced with plus signs. If you end up getting stuck later, that's where I'd fix

File: app/config/factionhall/events.json [hunk 3]
```diff
@@ -1,60 +1,52 @@
...
         {
-            "name": "The Rise and Fall of Bennett Braun by Evan Anderson",
-            "description": "Join Grey Faction director Evan Anderson to discuss The Rise and Fall of Bennett Braun.",
-            "room": "rotunda",
-            "start": "2022-08-20T21:00:00-04:00",
-            "end": "2022-08-20T22:00:00-04:00",
-            "image": "https://tstvhq.s3.amazonaws.com/backgrounds/factionhall/greyfaction/calendarhall.png"
+            "name": "Naloxone Training",
+            "description": "Minister Kal Set & Director of Sober Faction Outreach, Keleigh Rose, will host a Naloxone Training.",
+            "room": "jude-mirra",
```
Comment (kacycrider) [inline app/config/factionhall/events.json]: Jude-Mirra is not an active room

**Review by kacycrider — DISMISSED** [review]

**Review by LadyAbigor — COMMENTED** [review]
Updated event iimage

**Review by LadyAbigor — COMMENTED** [review]
- Updated calendar image
- Updated room names to match the events.json

**Review by kacycrider — APPROVED** [review]

**Pull request merged**